### DISCUSSION TITLE
calculation

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/python_book#17] [Announcements]
# calculation

数据是所有编程语言的核心，程序很多时候，就是为了把一些数据（输入）转换成另一些数据（输出）。对于 Python 来说，数据尤其重要，数据处理，机器学习等高度数据相关的工作都是 Python 最主要的领域。

https://py.qizhen.xyz/calculation

**Comment by ruanqizhen** [comment]
### 计算半径为 r 的圆的面积和周长
```python
r = 23
pi = 3.14

# 计算面积和周长
area = pi * r ** 2
circumference = 2 * pi * r
```

### 华氏度-摄氏度转换
```python
fahrenheit = 111.34
celsius = (fahrenheit - 32) * 5 / 9

# 反之： F = C * 9/5 + 32
```
